src/app/page/authorization/signup/signup.component.ts

```diff
@@ -50,12 +54,22 @@ export class SignUpComponent implements OnInit {
   }
 
   ngOnInit() {
+    const validators = [
+      Validators.required,
+      Validators.minLength(this.NAMES_MIN_LENGTH),
+      Validators.maxLength(this.NAMES_MAX_LENGTH),
+      Validators.pattern('[a-zA-Zа-яіІА-Я]*')
```

Comment:
Strange pattern detected. 

As a user I would like to use arabic letters in my name.